README.md

```diff
@@ -29,9 +29,21 @@
 - IntelliSense (completions) will be provided as you write your scripts
 - Select the script and press `CMD+"` (`CTRL+"` on Windows and Linux) to see the results
 
-## Sign in to Azure to view your Cosmos DB Accounts
+## Managing Azure Subscriptions
 
-![Azure Sign In](resources/signin.gif)
+Sign in to Azure to view your Cosmos DB Accounts

```

Comment:
Remove this line - looks like its leftover from the old header

Reply:
done